### PR TITLE
WIP: Teams diff with master

### PR DESCRIPTION
**Pull request by gmegan**
PR to track differences in teams and master spec branch. Do not merge.

### CODE REVIEW

**Comment by gmegan** [comment]
Current PDF:

[main_spec.pdf](https://github.com/gmegan/specification/files/2558314/main_spec.pdf)

**Comment by gmegan** [comment]
Current PDF
[main_spec.pdf](https://github.com/gmegan/specification/files/2788433/main_spec.pdf)

**Comment by gmegan** [comment]
Current PDF
[main_spec.pdf](https://github.com/gmegan/specification/files/2841567/main_spec.pdf)

**Comment by gmegan** [comment]
Current PDF:
[main_spec.pdf](https://github.com/gmegan/specification/files/2885469/main_spec.pdf)

**Comment by naveen-rn** [comment]
Looks like examples are included - but not added in the doc.

**Comment by gmegan** [comment]
[main_spec.pdf](https://github.com/gmegan/specification/files/2963150/main_spec.pdf)

**Comment by gmegan** [comment]
[main_spec.pdf](https://github.com/gmegan/specification/files/2968367/main_spec.pdf)

**Comment by gmegan** [comment]
[main_spec.pdf](https://github.com/gmegan/specification/files/3003908/main_spec.pdf)

**Comment by gmegan** [comment]
[main_spec.pdf](https://github.com/gmegan/specification/files/3009244/main_spec.pdf)

File: content/shmem_teams_intro.tex [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+An \openshmem team is a set of \acp{PE} defined by calling a specific team
+split routine with a parent team argument and other arguments to
+specify how the parent team is to be split into one or more new teams.
+Any team created by a \FUNC{shmem\_team\_split\_*} routine can subsequently
+be used as the parent team for further calls to team split routines.
+A team persists and can be used for team-based routine calls
+until it is destroyed by \FUNC{shmem\_team\_destroy}.
+
+Every team must have a least one member. Any attempt to create a team over an
```
Comment (wrrobin) [inline content/shmem_teams_intro.tex]: `a` => `at`

File: content/shmem_teams_intro.tex [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+including collective routines (see Section~\ref{subsec:coll}), include team
+creation routines.  While \openshmem routines are thread-safe as
+per threading model (see section \ref{subsec:thread_support}),\openshmem
+teams objects are not themselves thread-safe. It is the responsibility
+of the application to ensure that there are no simultaneous collective
+routines operating on the same \openshmem team on a given \ac{PE}.
+
+\subsubsection*{Team Objects and Collective Ordering across \acp{PE}}
+
+In \openshmem, a team object encapsulates resources uses to communicate
```
Comment (wrrobin) [inline content/shmem_teams_intro.tex]: `uses` => `used` ?

**Comment by gmegan** [comment]
[main_spec.pdf](https://github.com/gmegan/specification/files/3370697/main_spec.pdf)

**Comment by gmegan** [comment]
[main_spec.pdf](https://github.com/gmegan/specification/files/3483893/main_spec.pdf)

**Comment by BryantLam** [comment]
page 30, line 47 - missing "to"
~~page 34, line 18 - the "otherwise ..." clause is ambiguously referring to the "when" or "if" clause~~ This is probably fine.
Annex Deprecation: For completeness in the event the routines are ever removed from the main spec, the `shmem_*_to_all` routines need to be listed in the Table and the Rationale. See deprecated atomics as an example (Annex E, 2.2).

File: content/shmem_team_config_t.tex [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+\begin{apiarguments}
+  None.
+\end{apiarguments}
+
+
+\apidescription{
+  A team configuration argument acts as an input
+  \FUNC{shmem\_team\_split\_*} routines.
+  It specifies the requested capabilities of the team to be
```
Comment (minsii) [inline content/shmem_team_config_t.tex:21]: Should it be an input **of**  \FUNC{shmem\_team\_split\_*} routines ?

**Pull request merged**